critiquebrainz/frontend/templates/moderators/moderators.html

```diff
@@ -5,14 +5,14 @@
 {% block content %}
 <div class="container">
```

Comment:
Other pages have all style items wrapped into a block with a unique ID. This is done to prevent conflicts.

You can add `id="moderators-list"`, for example, and wrap all the items into that id:
```less
#moderators-list {
  ...
}
```